runtime/lua/vim/treesitter.lua

```diff
@@ -187,15 +187,20 @@ end
 ---@returns string
 local function buf_range_get_text(buf, range)
   local start_row, start_col, end_row, end_col = M._range.unpack4(range)
+  local insert_nl = false
   if end_col == 0 then
     if start_row == end_row then
       start_col = -1
       start_row = start_row - 1
     end
     end_col = -1
     end_row = end_row - 1
+    insert_nl = true
   end
   local lines = api.nvim_buf_get_text(buf, start_row, start_col, end_row, end_col, {})
+  if insert_nl then
+    table.insert(lines, "")
```

Comment:
The double quotes are failing the linter test. Make it single quotes.

https://github.com/neovim/neovim/actions/runs/9142308887/job/25137718542?pr=28841

![image](https://github.com/user-attachments/assets/4aa18831-3e25-4e65-a85e-c2b9bdf8ea29)